src/store/actions/transactions.test.js

```diff
@@ -63,5 +90,198 @@ describe('actions: transactions', () => {
         data: expectedAction,
       });
     });
+
+    it('should dispatch getTransactionsSuccess action if resolved and default argument values are used', async () => {
+      const transactionData = {
+        address: '15626650747375562521L',
```

Comment:
we should use new lisk address, for now use this `lsks6uckwnap7s72ov3edddwgxab5e89t6uy8gjt6` and we need an issue to update all other fixtures use proper address etc..

Reply:
Fixed